ppr-ui/src/components/tables/common/TableRow.vue

```diff
@@ -672,6 +694,10 @@ export default defineComponent({
       return item.registrationType === APIRegistrationTypes.REPAIRERS_LIEN
     }
 
+    const isTransAffi = (type: ApiTransferTypes): boolean => {
```

Comment:
This could be moved to useTransferOwners composable, along with other types.

Reply:
Indeed, updating 👍